app/index.js

```diff
@@ -6,9 +6,31 @@ const yeoman = require('yeoman-generator');
 const _s = require('underscore.string');
 
 module.exports = yeoman.Base.extend({
+	constructor: function () { // eslint-disable-line babel/object-shorthand
```

Comment:
I tried that, and something complained. I think maybe Babel? Something about " `constructor () {...` is not a constructor " the output didn't make a lot of sense